diff --git a/app/include/zmk/hid.h b/app/include/zmk/hid.h
index 0f9cfe2e..01104d1c 100644
--- a/app/include/zmk/hid.h
+++ b/app/include/zmk/hid.h
@@ -52,7 +52,7 @@ static const uint8_t zmk_hid_report_desc[] = {
     HID_INPUT(0x02),
 #elif IS_ENABLED(CONFIG_ZMK_HID_REPORT_TYPE_HKRO)
     HID_LOGICAL_MIN8(0x00),
-    HID_LOGICAL_MAX8(0xFF),
+    HID_LOGICAL_MAX16(0xFF, 0x00),
     HID_USAGE_MIN8(0x00),
     HID_USAGE_MAX8(0xFF),
     HID_REPORT_SIZE(0x08),
