diff --git a/app/include/zmk/hid.h b/app/include/zmk/hid.h
index 01104d1c..1a9ace55 100644
--- a/app/include/zmk/hid.h
+++ b/app/include/zmk/hid.h
@@ -13,7 +13,7 @@
 #include <dt-bindings/zmk/hid_usage.h>
 #include <dt-bindings/zmk/hid_usage_pages.h>
 
-#define ZMK_HID_KEYBOARD_NKRO_MAX_USAGE HID_USAGE_KEY_KEYPAD_EQUAL
+#define ZMK_HID_KEYBOARD_NKRO_MAX_USAGE 0xEF
 
 #define COLLECTION_REPORT 0x03
 
